docs/napari-plugin-guide.md

```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
-
- 
-
-> *<u>Extinction 100-200:</u>* Good Extinction. These are the typical values we get on our microscopes.
-
- 
-
-> <u>*Extinction 200+:*</u> Phenomenal. Indicates a very well-aligned and clean light path and high sensitivity of the system.
-
-
-
-### Load Calibration*
+* Extinction 200+: Excellent. Indicates a very well-aligned and clean light path and high sensitivity of the system.
 
+### Optional: Load Calibration
 If a user wants to use a previous calibration to acquire new data, then the "Load Calibration" button can be pressed.  It will direct you to select a *calibration_metadata.txt* file and these settings will be automatically updated in MicroManager.  recOrder will also collect a few images to update the extinction ratio to reflect the current conditions.  Once this has finished, a user can now acquire data as they normally would.  
 
+This is quite useful if micromanager and/or  recOrder crashes. If the sample and imaging setup haven't changed, it is safe to use a past calibration. Otherwise, if a new sample is used or some microscope components are changed, we recommend performing a new calibration.
 
-
-*This is quite useful for micromanager crashes and potential recOrder crashes.  If nothing about the sample / imaging setup has changed, it is safe to use a past calibration.  Otherwise, if a new sample is used or some microscope components are changed, it is recommended to perform a new calibration.
-
-
-
-### Calculate Extinction
-
+### Optional: Calculate Extinction
 This is a useful feature to see if the extinction level varies as you move around the sample.  Sometimes there can be local variations present in the sample which can cause slightly different perturbations to the polarization state.  If the extinction level varies dramatically across the sample, it is worthwhile to calibrate and acquire background images as close to the area in which you will be imaging as possible.
 
-
-
-### Capturing Background
-
+### Capture Background
 The next important step in the calibration process. This will later serve in reconstruction to correct for any local and global background anisotropy. 
 
 ![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/cap_bg.png)
 
 Choose the name of the folder to which to save your background images (will be placed into the save directory chosen at the beginning). Choose the number of images to average, 20 and below is generally good.  The background image results will then be displayed in the napari window.  It is normal to see some level of background retardance and orientation bias-- this can be corrected with the background correction step in reconstruction.  Examples of this display are below.
 
-*NOTE: If you wish the capture multiple background sets, please change the folder name in between captures as specifying the same name will overwrite previous data.*
-
 ![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/bg_example.png)
 
 ### Advanced Tab
-
```

Comment:
Wrapping headings with line breaks is recommended by [markdownlint](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md022). But the parser that has issues with this does not comply to CommonMark specs, which GitHub adopted.